lib/assessment.md

```diff
@@ -41,7 +50,10 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
+#contentPane {}
+.important .red {}
```

Comment:
This will give you the _union_ of the two classes ('important', 'red', and 'important and red'); we were looking for the _intersection_ of the two classes ('important and red' only). However, the prompt was a little ambiguous, so I'll accept this answer.